.eslintrc

```diff
@@ -35,6 +36,8 @@
     "consistent-return": "error",
     "curly": ["error", "all"],
     "default-case": "error",
+    "eol-last": "error",
```

Comment:
<!-- ideas that popped up from reviewing -->
**💡 thought:**
Should we also use [no-multiple-empty-lines](https://eslint.org/docs/latest/rules/no-multiple-empty-lines) together with it? To enforce just one multiple line.

Reply:
Good idea, implemented.